docs/en/operations/utils/clickhouse-benchmark.md

```diff
@@ -0,0 +1,145 @@
+- `--confidence=N` — Level of confidence for T-test. Possible values: 0 (80%), 1 (90%), 2 (95%), 3 (98%), 4 (99%), 5 (99.5%). Default value: 5.
+- `--cumulative` — Printing cumulative data instead of data per interval.
+- `--database=DATABASE_NAME` — ClickHouse database name. Default value: `default`.
+- `--json=FILEPATH` — JSON output. When the key is set, `clickhouse-benchmark` outputs a report to the specified JSON-file.
+- `--user=USERNAME` — ClickHouse user name. Default value: `default`.
+- `--password=PSWD` — ClickHouse user password. Default value: empty string.
+- `--stacktrace` — Stack traces output. When the key is set, `clickhouse-bencmark` outputs stack traces of exceptions.
+- `--stage=WORD` — Query processing stage at server. ClickHouse stops query processing and returns answer to `clickhouse-benchmark` at the specified stage. Possible values: `complete`, `fetch_columns`, `with_mergeable_state`. Default value: `complete`.
+- `--help` — Shows the help message.
+
+If you want to apply some [settings](../../operations/settings/index.md) for queries, pass them as a key `--<session setting name>= SETTING_VALUE`. For example, `--max_memory_usage=1048576`.
+
+## Output
+
+
+By default, `clickhouse-benchmark` reports for each `--delay` interval.
+
+Example of the report:
+
+```text
+Queries executed: 10.
+
+localhost:9000, queries 10, QPS: 6.772, RPS: 67904487.440, MiB/s: 518.070, result RPS: 67721584.984, result MiB/s: 516.675.
+
+0.000%		0.145 sec.	
+10.000%		0.146 sec.	
+20.000%		0.146 sec.	
+30.000%		0.146 sec.	
+40.000%		0.147 sec.	
+50.000%		0.148 sec.	
+60.000%		0.148 sec.	
+70.000%		0.148 sec.	
+80.000%		0.149 sec.	
+90.000%		0.150 sec.	
+95.000%		0.150 sec.	
+99.000%		0.150 sec.	
+99.900%		0.150 sec.	
+99.990%		0.150 sec.	
+```
+
+In the report you can find:
+
+- Number of queries in the `Queries executed: ` field.
+- Status string containing (in order):
+
+    - Endpoint of ClickHouse server.
+    - Number of processed queries.
+    - QPS. Queries Per Second server metric for the period (`--delay`) of reporting.
+    - RPS. Rows Per Second server metric for the period (`--delay`) of reporting.
+    - MiB/s. Megabytes Per Second server metric for the period (`--delay`) of reporting.
+    - result RPS. Rows Per Second server metric for the whole period.
+    - result MiB/s. Megabytes Per Second server metric for the whole period.
+
+- Percentiles of queries execution time.
+
+
+## Comparison mode
+
+`clickhouse-benchmark` can compare performances for two running ClickHouse servers. To use the comparison mode, specify endpoints of both servers by two pairs of `--host`, `--port` keys. `clickhouse-benchmark` establishes connections to both servers, then sends queries selecting destination server for each query randomly. The results are shown for each server separately.
```

Comment:
BTW it doesn't send queries concurrently. Only one server is loaded at the single moment of time to avoid interference. And the choice of the server is alternated randomly.